go.mod

```diff
@@ -75,3 +75,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.4.0
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/longhorn/longhorn-instance-manager v0.0.0-20210729081215-50c310f97378 => github.com/derekbit/longhorn-instance-manager v0.0.0-20220411065951-c052142954b7
```

Comment:
Why using replace instead of updating the required dep directly?

Reply:
Oh, we are still waiting for https://github.com/longhorn/longhorn-instance-manager/pull/108 😄